src/gui/instrument/InstrumentTrackWindow.cpp

```diff
@@ -264,9 +264,11 @@ InstrumentTrackWindow::InstrumentTrackWindow( InstrumentTrackView * _itv ) :
 	m_tabWidget->addTab(m_tuningView, tr("Tuning and transposition"), "tuning_tab", 5);
 
 	// setup piano-widget
+	auto pianoLayout = new QHBoxLayout();
 	m_pianoView = new PianoView( this );
```

Comment:
@regulus79 wrote:

> I would like to get someone's opinion on whether I am managing the layout pointer correctly. Passing this as the parent for the layout caused warnings, since that automatically tries to set the layout as the default layout for the parent. Should I leave it as is and add an explicit call to setParent(this) to make sure this is responsible for deleting the layout?